README.md

```diff
@@ -96,7 +96,8 @@ As of now, these are the supported resources:
     -f,--full                           By default, a delta sync runs using
                                         last-sync-timestamp logic. Use this
                                         flag to run a full sync. i.e. sync
-                                        the entire data set.
+                                        the entire data set. This option must
+                                        be added after `-s` option.
     -h,--help                           Print help information.
     -r,--runnerName <arg>               Choose a name for the running sync
```

Comment:
What about this option? I think this also should come only after -s.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/410/commits/05d19b30007178a17d817f003fde377395285b8e